### PR TITLE
Updates to PHSA

### PR DESCRIPTION
**Pull request by mtpultz**
* Changed validation message to AlertComponent to match radio buttons since mat-error doesn't work with checkboxes and radio buttons
* Updated PHSA URL to `/phsa-eforms`
* Clean up bindings in templates
* Use existing form validator
* Refactor to pipe replacing namespace
* Refactor PHSA to use FormState builders to decrease complexity of form state services

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export class PhsaLabtechRoutes {
-  public static PHSA_LABTECH = 'phsa-labtech';
+  public static PHSA_LABTECH = 'phsa-eforms';
```
Comment (AnaisHebert) [inline prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts:2]: should we change the variable name to `PHSA_EFORMS`?

File: prime-angular-frontend/src/app/modules/phsa-labtech/shared/services/phsa-labtech-form-state.service.ts [hunk 2]
```diff
@@ -28,21 +29,23 @@ export class PhsaFormStateService extends AbstractFormStateService<PhsaEnrollee>
   }
 
   public get json(): PhsaEnrollee {
-    const { phone, phoneExtension, email } = this.demographicsForm.getRawValue();
-    // TODO: Needs to get value (code) of selected checkboxes
-    const { partyTypes } = this.availableAccessForm.getRawValue();
+    // const accessCode = this.accessCodeFormState.json;
```
Comment (AnaisHebert) [inline prime-angular-frontend/src/app/modules/phsa-labtech/shared/services/phsa-labtech-form-state.service.ts:32]: should this be here?

Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/phsa-labtech/shared/services/phsa-labtech-form-state.service.ts:32]: I don't know.  Was removed when I was on vacation

File: prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts (hunk 1 above)
Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/phsa-labtech/phsa-labtech.routes.ts:2]: Sure, but then I'd want to rename the entire module.  So we could do that.

**Review by AnaisHebert — APPROVED** [review]
LGTM! 🎡

**Pull request merged**